### PR TITLE
added historic event - Barney Frank becomes second openly gay member …

### PR DESCRIPTION
**Pull request by aytvill**
…of Congress.

tiny contribution for Hacktoberfest

### CODE REVIEW

**Comment by aytvill** [comment]
should commit to issue #5

**Comment by aytvill** [comment]
more addition to issue #5

**Comment by aytvill** [comment]
last item could be good for issue #4

File: words_dir/tweet_adjectives.py
Comment (VickiLanger) [inline words_dir/tweet_adjectives.py:1]: Love it!

**Review by VickiLanger — APPROVED** [review]
Thanks for your additions LGTM

**Pull request merged**